app/controllers/api/v1/keywords_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class KeywordsController < ApplicationController
+      include ::Pagy::Backend
+      include API::V1::Pagy::JSONAPIConcern
+
+      def index
+        pagy, keywords_list = pagy(keywords)
+
+        if keywords.any?
+          render json: KeywordSerializer.new(keywords_list, pagy_options(pagy)).serializable_hash
```

Comment:
Can we also try to remove `.serializable_hash`

Reply:
Yes @ankitkalia1195, you are right! It's working! 🚀